cdc-mi/1.0.0wso2v1/pom.xml

```diff
@@ -0,0 +1,248 @@
+<!--
```

Comment:
Component path change as mentioned in https://github.com/wso2/orbit/pull/995#issuecomment-1670613314

Reply:
is it okay to name it as Debezium since it contains Kafka and binlog connector dependencies as well?

Reply:
Yeah, that's fine. We are already doing similar in some of the other orbit bundles as well.

Reply:
Thanks for the clarification.  Changed the name accordingly